common/lib/api-client/middleware/request-timeout.js

```diff
@@ -2,6 +2,10 @@ module.exports = function requestTimeout(timeout) {
   return {
     name: 'request-timeout',
     req: payload => {
+      if (payload.res) {
+        return payload
+      }
+
       if (payload.req) {
```

Comment:
Is there a reason why we check for the presence of the `req` object? Surely the `payload` will always have it if passed by `devour`?

Reply:
Not off the top of my head, but I'd be inclined to say there was a reason.

Reply:
@solidgoldpig Figuring this out might be the solution to the req check vs return payload question! :)

Reply:
`Devour` always has a 'req` object on the `payload`. The only reason it wouldn't be there is if we explicitly deleted it.

I think it's overeager guard. Please, someone show me otherwise

NB. the example at https://github.com/twg/devour#your-first-middleware does not bother with the check